rosetta-source/src/main/rosetta/observable-event-type.rosetta

```diff
@@ -10,16 +10,15 @@ import cdm.observable.common.*
 
 type Trigger: <"Trigger point at which feature is effective.">
 
-    level number (0..1) <"The trigger level.">
-    levelPercentage number (0..1) <"The trigger level percentage.">
+    level PriceSchedule (0..2) <"The trigger level.">
```

Comment:
Do we really want differently named parameters between prod and dev?

Reply:
In prod, it is `levelPrice`, but in dev, it is `level`.  So anyone upgrading would need to change that.  If you make them the same, it would just work.

Reply:
yes, that what we discussed this morning to unify it to a single level elemt, but I see your point Hugo, we can put the levelPrice element and delete level/levelPercentage then.

Reply:
I just have a chat with Anna and Marc and we concluded that it would be better to use ` level PriceSchedule (0..2)` since we have also a RR in DRR that would be using that level element. We can delete CDM Prod release and keep this one, would be that ok for you Hugo?

Reply:
Currently, DRR (Prod and Dev) both use CDM Prod, so if you only make the change in CDM Dev then it can't be picked up in DRR.

Reply:
I discussed with @gopazoTH  offline, and understood that the intention is to not release on CDM Prod